### ISSUE TITLE
する verbs 送り仮名 is highlighted together with 漢字

### ISSUE THREAD

**Issue by Konstantin-Glukhov** [birchill/10ten-ja-reader#1260]
If highlighting 送り仮名 after する verbs is an intended feature, then maybe there should be an option to turn it on/off? I see at least two problems with highlighting 送り仮名 by default:

1. if highlight is copied, then it contains extra stuff that needs to be deleted
2. the extra highlight takes away the attention from the entry itself

**Comment by SaltfishAmi** [comment]
Rather than "intended feature" of the extension, I would say it's a characteristic of the 送り仮名 support of the browser - if you highlight them manually, with your cursor one character at a time, you will also find out that the 送り仮名 is positioned **in between** the 漢字 and trailing する. So I assume that adding such option may require additional work.

**Comment by Konstantin-Glukhov** [comment]
Rikaichan did not behave this way.

**Comment by SaltfishAmi** [comment]
Edit - I've got it wrong: I thought it was 振り仮名. Absolutely my bad.

What do you mean by 送り仮名 after する verbs? I would like an example because it doesn't make much sense to me.

**Comment by Konstantin-Glukhov** [comment]
Maybe I am using the wrong term. It is the する itself that is highlighted together with the kanji term, e.g. if the mouse cursor is placed over 拝見させて, then させて is highlighted together with 拝見, in other words the whole 拝見させて is highlighted.

**Comment by SaltfishAmi** [comment]
Oh yes. That was an intended feature. See also: #180

**Comment by Konstantin-Glukhov** [comment]
I believe this should be an option for the reasons I stated earlier.

**Comment by birtles** [comment]
I agree with de-emphasizing the する part (it's not really okurigana) and have a plan to do that.

The copying should just be copying the dictionary entry, however, so it already shouldn't include the する part. Is that not the case?

**Comment by Konstantin-Glukhov** [comment]
Yes, copying works fine via app shortcut, but I just learned that Ctrl-C is disabled on the app pop-up (In my first note I assumed it works). I think Ctrl-C should work too.

**Comment by Konstantin-Glukhov** [comment]
On unrelated subject of terminology: what is hiragana following a verb starting with Kanji called? Okurigana. By analogy the suru part can also be called okurigana, though technically it's not.
